supervisor/host/network.py

```diff
@@ -74,11 +69,17 @@ def dns_servers(self) -> List[str]:
 
     async def check_connectivity(self):
         """Check the internet connection."""
+        if HostFeature.NETWORK not in self.sys_host.supported_features:
```

Comment:
Let's just check `self.sys_dbus.network.is_connected`, the supported features property cost a lot with all the checks